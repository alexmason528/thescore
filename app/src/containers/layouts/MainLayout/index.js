import React from 'react'
import { Layout } from 'antd'

const { Header, Content, Footer } = Layout

const MainLayout = ({ children }) => (
  <Layout className="page">
    <Header className="header"></Header>
    <Content className="content">{children}</Content>
    <Footer className="footer">The Score</Footer>
  </Layout>
)
export default MainLayout
