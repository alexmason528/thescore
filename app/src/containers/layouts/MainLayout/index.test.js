import React from 'react'
import { Button } from 'antd'
import { shallow } from 'enzyme'

import MainLayout from './index'

describe('MainLayout', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <MainLayout>
        <Button>Click</Button>
      </MainLayout>
    )
    expect(wrapper).toMatchSnapshot()
  })
})
