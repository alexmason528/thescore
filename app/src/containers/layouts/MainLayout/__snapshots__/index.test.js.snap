// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout renders correctly 1`] = `
<Layout
  className="page"
>
  <Header
    className="header"
  >
    <img
      alt="Logo"
      className="logo"
      src="logo.svg"
    />
  </Header>
  <Content
    className="content"
  >
    <Button
      block={false}
      ghost={false}
      htmlType="button"
      loading={false}
    >
      Click
    </Button>
  </Content>
  <Footer
    className="footer"
  >
    <a
      href="https://www.thescore.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      theScore.com
    </a>
  </Footer>
</Layout>
`;
